src/parseTools.js

```diff
@@ -197,10 +197,20 @@ function isStructPointerType(type) {
   return !Compiletime.isNumberType(type) && type[0] == '%';
 }
 
+const POINTER_SIZE = MEMORY64 ? 8 : 4;
+const POINTER_BITS = POINTER_SIZE * 8;
+const POINTER_TYPE = 'i' + POINTER_BITS;
+
+const SIZE_TYPE = POINTER_TYPE;
```

Comment:
It's not obvious to me what this is. Is it for `size_t`? Why do we need it? (It's not used anywhere)